src/environment/mod.rs

```
@@ -321,6 +321,26 @@ pub async fn store_credentials_from_project(project: &Workspace) -> miette::Resu
     Ok(())
 }
 
+async fn create_file_if_not_exists(
```

This name doesnt tell me that read only filesystem errors are silently ignored. Can you clarify and add a commen?

---

Changed it, so it should be more clear now